src/modules/layout/components/Header.js

```diff
@@ -123,6 +124,24 @@ class Header extends React.PureComponent<PropsType, StateType> {
     this.getNavigation().navigate('ChangeLanguageModal')
   }
 
+  onShare = async () => {
+    const url = 'https://integreat-app.de/'
```

Comment:
I think it would be nice to get a link to the current page here, so if I'm on route 'testumgebung/en/welcome/' I want to get a link to 'https://integreat.app/testumgebung/en/welcome/'.  The path (if navigated to Categories) should be available in the store.
Also use our new domain 'https://integreat.app'.

Reply:
Yes we need to link to the current selected page. You probably need to merge in develop for that.